_aggregations/bucket/auto-interval-date-histogram.md

```diff
@@ -0,0 +1,377 @@
+
+As an example, index the following documents into an index:
+
+```json
+PUT blogs1/_doc/1
+{
+  "name": "Semantic search in OpenSearch",
+  "date_posted": "2022-04-17T01:00:00.000Z"
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT blogs1/_doc/2
+{
+  "name": "Sparse search in OpenSearch",
+  "date_posted": "2022-04-17T04:00:00.000Z"
+}
+```
+{% include copy-curl.html %}
+
+First, run an aggregation without specifying a time zone: 
+
+```json
+GET /blogs1/_search
+{
+  "size": 0,
+  "aggs": {
+    "histogram": {
+      "auto_date_histogram": {
+        "field": "date_posted",
+        "buckets": 2,
+        "format": "yyyy-MM-dd HH:mm:ss"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response contains two 3-hour buckets, starting from midnight UTC on April 17, 2022:
+
+```json
+{
+  "took": 6,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "histogram": {
+      "buckets": [
+        {
+          "key_as_string": "2022-04-17 01:00:00",
+          "key": 1650157200000,
+          "doc_count": 1
+        },
+        {
+          "key_as_string": "2022-04-17 04:00:00",
+          "key": 1650168000000,
+          "doc_count": 1
+        }
+      ],
+      "interval": "3h"
+    }
+  }
+}
+```
+
+Now, specify a `time_zone` of `-02:00` 
+
+```json
+GET /blogs1/_search
+{
+  "size": 0,
+  "aggs": {
+    "histogram": {
+      "auto_date_histogram": {
+        "field": "date_posted",
+        "buckets": 2,
+        "format": "yyyy-MM-dd HH:mm:ss",
+        "time_zone": "-02:00"
+      }
+    }
+  }
+}
+```
+
+The response contains two buckets in which the start time is shifted by two hours and starts with 23:00 on April 16, 2022:
+
+```json
+{
+  "took": 17,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "histogram": {
+      "buckets": [
+        {
+          "key_as_string": "2022-04-16 23:00:00",
+          "key": 1650157200000,
+          "doc_count": 1
+        },
+        {
+          "key_as_string": "2022-04-17 02:00:00",
+          "key": 1650168000000,
+          "doc_count": 1
+        }
+      ],
+      "interval": "3h"
+    }
+  }
+}
+```
+
+When using time zones with daylight saving time (DST) changes, the size of buckets near the transition may differ slightly from the size of neighboring buckets. 
+{: .note}
```

Comment:
Above: Does singular work here? Otherwise I think it needs to be "the sizes of buckets that are near the transition may differ slightly from the sizes of neighboring buckets."